backend/handlers/handlers.go

```diff
@@ -685,6 +745,11 @@ func (handlers *Handlers) getConvertFromFiatHandler(r *http.Request) (interface{
 		unit = unit[2:]
 	}
 
+	btcCoin, _ := handlers.backend.Coin(coinpkg.CodeBTC)
+	if from == coinpkg.UnitBtc && btcCoin.(*btc.Coin).GetFormatUnit() == coinpkg.UnitSats {
+		fiatRat = new(big.Rat).Quo(fiatRat, new(big.Rat).SetFloat64(coinpkg.Btc2Sat))
```

Comment:
I see this `new(big.Rat).Quo(amountRat, new(big.Rat).SetFloat64(unitSatoshi))` line three times in this PR. Please move all of them to the btc package in a common function, ideally with a unit test.

Also, instead of SetFloat64(), maybe you can just use `big.NewRat()`? since the unitSatoshi const is not a float but an int.

Reply:
Done